core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -637,6 +637,7 @@ var ExplorationEditorMainTab = function() {
       NumericInput: 'Math',
       SetInput: 'Math',
       AlgebraicExpressionInput: 'Math',
+      MathEquationInput: 'Math',
       MathExpressionInput: 'Math',
```

Comment:
Is `AlgebraicExpressionInput` replacing `MathExpressionInput`? If so, should we remove `MathExpressionInput` from here? If not, then don't the changes in `extensions.js` mean those tests don't cover `MathExpressionInput` anymore?

Reply:
Yes, the new interactions replace the old one. I've removed it from here.